package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "bonita-documentation-site",
+  "version": "0.0.1",
```

Comment:
Add licence field

Reply:
Yes this is missing, I expected to set it as part of #10 when we will have a clear answer from Bonitasoft, but maybe we already have one